.github/workflows/build-windows-selfhosted.yml

```diff
@@ -68,7 +68,13 @@ jobs:
       - run: unzip fstar*.zip
         shell: bash
 
+      - name: Check out Hello example
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: examples\hello\Hello
```

Comment:
This file is a one-liner, can we just create it here with `cat` or something, to avoid this job depending on `examples/hello`? We do this in linux/mac, unsure if it would work in Windows:
```
echo -e "module A\nopen FStar.Mul\nlet _ = assert (forall x. 1 + x*x > 0)" > A.fst
```